lib/pliny/error_reporters/rollbar.rb

```diff
@@ -18,7 +18,7 @@ def notify(exception, context:, rack_env:)
       private
 
       def fetch_scope(context:, rack_env:)
-        scope = {}
+        scope = { custom: context }
```

Comment:
Weakly held opinion, but is there a reason we don't call this `context`? Does that name hold some significance to Rollbar?